docs/development/i18n.rst

```diff
@@ -298,6 +298,34 @@ Note that the new index will not be used right away, some workers may
 still have the old index open. Rebooting the machine is an option,
 waiting for a few hours is another option.
 
+Release management
+------------------
+
+Two weeks before the release: update
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The new and updated strings are uploaded to Weblate. This is done late in the SecureDrop release cycle so translators get less notifications. It would be inconvenient if there were hundreds of strings needing attention. But SecureDrop is small and it is ok to postpone notifications.
+
+* Merge develop into the Weblate fork
+* Post an announcement `to the translation section of the forum <https://forum.securedrop.club/c/translations>`__ (see `an example  <https://forum.securedrop.club/t/10-securedrop-strings-need-work-february-2018-feature-freeze/449>`__)
+* Add a prominent Weblate whiteboard announcement that reads `The X.Y.Z deadline is MM DD, YY midnight. String freeze will be in effect MM DD, YY midnight.`
+* Create a pull request for every source string suggestion coming from translators
+
+One week before the release: string freeze
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Merge develop into the Weblate fork
```

Comment:
With this workflow, in a 0.6-like scenario where the release branch is created two weeks before the release, wouldn't _any_ string changes made in `develop` need to be backported to the release branch? Or am I missing something?

Reply:
Good point! This is exactly what happened with 0.6. Fixing accordingly.